Sources/Prometheus/PrometheusCollectorRegistry.swift

```diff
@@ -47,11 +47,33 @@ public final class PrometheusCollectorRegistry: Sendable {
         }
     }
 
+    // Note: In order to support Sendable, need to explicitely define the types.
```

Comment:
slightly weird note; we're not marking those Sendable at all? Remove the note?
```suggestion
```

Reply:
Was intended to avoid someone proposing something like I had at the beginning, see https://github.com/swift-server/swift-prometheus/pull/131#discussion_r2252882330. Below would not work here:

```
private struct MetricGroup<T> {
    var metricsByLabelSets: [LabelsKey: T]
}
```

It may not be a good note; still remove or adjust it?

Reply:
Yeah, something like that. I proposed a patch below; we can converge those types a bit more and annotate them as Sendable as well: https://github.com/swift-server/swift-prometheus/pull/131#issuecomment-3222165204

Reply:
Ah, I see where I was wrong with my initial attempt. Awesome, glad this suboptimal note got us to re-review this part!